### ISSUE TITLE
Other New 3DS Functions Suggestion

### ISSUE THREAD

**Issue by PRGilland** [Roberto-Nessy/OoT3D_Standalone_Free_Cam#5]
This mod is a godsend, thank you. 

This is probably a tall order, but I was wondering if it is possible to add in other features from the New 3DS, such as the ZL and ZR buttons? And if so, have them access touch screen functions as extra button layers, the same way Leoetlino's Project Restoration does for Majora's Mask 3D? Perhaps other bindings such as Start for the item menu, D-pad, etc and so forth.

**Comment by FierceDIO** [comment]
On the same note, I'd like to issue suggestions for the button mapping would this be added to the mod.
This is to have consistency between OoT3D and MM3D.

ZL : First person
ZR+A : Ocarina
ZR+X : Item I
ZR+Y : Item II
Select : Items
ZR+Start : Gear
ZR+Select : Map
D-Pad Right : Ocarina/Navi

The rest of the D-Pad would be used to change equipment without going into the Gear or Items menus.

Child;
D-Pad Left : Swap Shield
D-Pad Down : Mask
(IDK what D-Pad Up could be)

Adult;
D-Pad Left : Swap Sword
D-Pad Down : Iron Boots
D-Pad Up : Hover Boots
ZR+D-Pad Left : Swap Shield
ZR+D-Pad Down : Goron Tunic
ZR+D-Pad Up : Zora Tunic

**Comment by PRGilland** [comment]
Also being able to swap arrow elements with the ZL button, like project restoration and Wind Waker. There is the slight issue of them consuming magic when readied, rather than fired, though.

Personally, I wouldn't put equipment on the d-pad, just because you're not really required to swap often, if ever, but that's just me. I've seen mods for the original OoT would make the d-pad just the boots, ocarina, and lens of truth. At least in cases where you couldn't set your own items. Young Link could probably have shields in place of the boots though.

I'm working on a Steam input configuration that simulates some of Project Restoration's button layers using Citra's touchscreen mappings, as well as a binding setting for [Rehid](https://github.com/hax0kartik/rehid). Can't do anything about the d-pad though.

**Comment by BakerIsVeryHungry** [comment]
Agreed. I was gonna make a post suggesting this, but it seems somebody jumped on it before I did. Love to hear it! Great job with the mod so far nonetheless. It's been pretty great so far.
